scripts/terraform/iac.sh

```diff
@@ -1,6 +1,8 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
 set -e
-terraform init
+
+terraform init -upgrade=true
```

Comment:
always good to upgrade for loose version pinning to ensure latest match is pulled